qsdsan/utils/parsing.py

```diff
@@ -156,7 +156,8 @@ def get_stoichiometric_coeff(reaction, ref_component, components, conserved_for,
         coeff_dct = symbolize(coeff_dct, components, conserved_for, parameters)
         coeff = dct2list(coeff_dct, components)
     if ref_component:
-        normalize_factor = abs(coeff[components._index[ref_component]])
+        try: normalize_factor = abs(coeff[components.index(str(ref_component))])
```

Comment:
Do we still need the breakpoint here?

Reply:
Took it out in 48bab7283dd35b9cc56cc97f3715c9241f1aef97, thanks!